src/metabase/models/field_values.clj

```diff
@@ -175,6 +175,12 @@
 ;;; |                                                  Utils fns                                                     |
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
+(defn inactive?
+  "If FieldValues have not been accessed recently they are considered inactive."
+  [field-values]
+  (and field-values (t/before? (:last_used_at field-values)
+                               (t/minus (t/offset-date-time) (t/days 14)))))
```

Comment:
nit: I think we should move `t/days 14` to a variable and maybe have a docstring for it.
like we have for `advanced-field-values-max-age` var in this ns.